input/templates/partials/form/errors.json

```diff
@@ -0,0 +1,7 @@
+{
+  "globalErrors" : [
```

Comment:
I've created these errors and messages examples with the proper structure. From now on we have to make sure they are always present in all form JSON data, so that we make sure they are displayed. Otherwise we always forget about them in half the forms.